app/views/ckey2bibframes/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Ckey to Bibframe2 Conversion</h1>
+<div class="btn-group">
+  <%= link_to 'Do another conversion', new_ckey2bibframe_path, class: 'btn btn-md btn-default btn-full' %>
+</div>
+<h3>MarcXML</h3>
+<pre>
+  <%= @ckey2bibframe.marc21_to_xml %>
+</pre>
+<h3>Bibframe2</h3>
+<pre>
+  <%= @ckey2bibframe.marcxml_to_bibframe %>
+</pre>
+<div class="btn-group">
+  <%= link_to 'Do another conversion', new_ckey2bibframe_path, class: 'btn btn-md btn-default btn-full' %>
+</div>
```

Comment:
I added a `conversion_button` helper method to `application_helper.rb` but I still have to call the helper method twice on the view.